### PR TITLE
Prepare v0.62.0 release

### PR DESCRIPTION
**Pull request by ashutosh-narkar**

### CODE REVIEW

**Comment by ashutosh-narkar** [comment]
On the Netlify failure, the log says 
```
3:43:16 PM: # github.com/prometheus/common/model
3:43:16 PM: vendor/github.com/prometheus/common/model/metric.go:363:33: undefined: strings.CutPrefix
3:43:16 PM: note: module requires Go 1.20
```

I'll investigate that.

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,43 @@
...
+
+### Docs + Website + Ecosystem
+
+- Add OpaDotNet to ecosystem projects ([#6554](https://github.com/open-policy-agent/opa/pull/6554)) authored by @me-viper
+- Add updated logos for Permit.io and OPAL ([#6562](https://github.com/open-policy-agent/opa/pull/6562)) authored by @danielbass37
+- docs: Update description of the url path usage when accessing values inside object and array documents for v1/data GET and POST ([#6567](https://github.com/open-policy-agent/opa/pull/6567)) authored by @ashutosh-narkar
+- docs: Use `application/yaml` instead of `application/x-yaml` as the former is now a recognized content type ([#6565](https://github.com/open-policy-agent/opa/pull/6565)) authored by @anderseknert
+
+### Miscellaneous
+- Add Elastic to ADOPTERS.md ([#6568](https://github.com/open-policy-agent/opa/pull/6568)) authored by @orouz
```
Comment (johanfylling) [inline CHANGELOG.md:33]: This doesn't belong under the `Docs + Website + Ecosystem` section?

**Review by johanfylling — APPROVED** [review]
👍

**Comment by ashutosh-narkar** [comment]
Seems to be an issue with the Go version in the Netlify runner. From the logs

```
3:40:35 PM: Installing Go version 1.22 (requested 1.22)
3:40:35 PM: Failed to install Go version "1.22"
3:40:36 PM: Continue with existing version "1.19.13"
```

File: CHANGELOG.md (hunk 1 above)
Comment (ashutosh-narkar) [reply CHANGELOG.md:33]: Adopter file changes have been in the `Miscellaneous` section sometimes and `Docs` in some cases so just added in `Miscellaneous`.

File: .go-version [hunk 2]
```diff
@@ -1 +1 @@
-1.22
+1.21.5
```
Comment (ashutosh-narkar) [inline .go-version:1]: This change is to make Netlify builds pass as there seem to be failures while installing Go v1.22  and then Go v1.19 is used as a fallback. The minimum version needed to build OPA to Go v1.20

**Comment by srenatus** [comment]
OK let's rebase this -- netlify works with 1.22 now as of #6605

**Comment by ashutosh-narkar** [comment]
> OK let's rebase this -- netlify works with 1.22 now as of #6605

Done!

**Review by srenatus — APPROVED** [review]
LGTM Thanks!

**Pull request merged**